### ISSUE TITLE
Question regarding the FFHQ dataset

### ISSUE THREAD

**Issue by utkarshojha** [rosinality/stylegan2-pytorch#88]
Hi, thanks for sharing the implementation!
I wanted to confirm if you've trained your model on the face-aligned version of FFHQ dataset or the raw (default FFHQ) one. In the official paper, the authors mention that they align the faces as part of the pre-processing step.

**Comment by rosinality** [comment]
I have used aligned and cropped (1024×1024) version of ffhq.

**Comment by CrossLee1** [comment]
The FFHQ dataset consists of 70,000 images.
but in README, it said

> Sample from FFHQ. At 110,000 iterations. (trained on 3.52M images)

Is there a newer version of FFHQ dataset?

**Comment by utkarshojha** [comment]
I suppose 110,000 iterations mean that the discriminator has seen (110,000*32 = 3.52M) images, where 32 is the batch size. It has nothing to do with how many images were actually there in the dataset.

**Comment by rosinality** [comment]
Yes, it is the total number of the images during the training, including duplicates.  The authors of the StyleGAN2 uses that conventions to indicate training iterations.

**Comment by liuliuliu11** [comment]
thanks for sharing the implementation!
Currently, I want to run train.py using a small part of the fftq dataset successfully in my notebook(with NVIDIA GeForce MX250
), so: set the parameter ‘ iter’ value to 100, and I have created lmdb datasets. I want to ask: how to determine the value about "torch.distributed.launch --nproc_per_node=N_GPU --master_port= PORT" ？In addition, does the code support PyTorch 1.5 and CUDA 10.2？
Looking forward to your reply！

**Comment by rosinality** [comment]
@liuliuliu11

N_GPU is number of gpus you want use for training. PORT is arbitrary port that available, like 8090.

You can use pytorch 1.5 and cuda 10.2. Actually I am using that environments.
